docs/testing-teal.md

```diff
@@ -117,6 +146,12 @@ Now, we will execute a transaction with stateful TEAL (which increments a global
 
 Complete test can be found [here](../packages/runtime/test/integration/stateful-counter.ts).
 
+## Best Practices
+- Follow Flow of Testing section to set up your tests.
+- Keep each test independent of each other.
+- Use `beforeEach` and `AfterEach` functions to set clear boundaries while testing.
```

Comment:
Can we add `beforeAll`, `afterAll` as well?

Reply:
yes